docs/api.md

```diff
@@ -299,7 +300,7 @@ those common validations are defined here.
 * `HEX_STRING`: `/^(?:[a-fA-F0-9]{2})+$/`
 * `URLSAFEBASE64`: `/^[a-zA-Z0-9-_]*$/`
 * `BASE_36`: `/^[a-zA-Z0-9]*$/`
-* `URL_SAFE_BASE_64`: `/^[A-Za-z0-9_-]*$/`
+* `URL_SAFE_BASE_64`: `/^[A-Za-z0-9_-]+$/`
```

Comment:
Also, should we get into the habit of opening a separate PR for updating the docs if they haven't been updated for earlier changes? Presumably those PRs would generally get reviewed/merged quickly and it would keep unrelated changes out of other PRs.